src/R/Editor/Impl/Completion/Providers/PackageFunctionCompletionProvider.cs

```diff
@@ -60,8 +60,12 @@ public class FunctionCompletionProvider : IRCompletionListProvider {
                 IEnumerable<INamedItemInfo> functions = pkg.Functions;
                 if (functions != null) {
                     foreach (INamedItemInfo function in functions) {
-                        bool? isSnippet = infoSource?.IsSnippet(function.Name);
-                        if (!isSnippet.HasValue || !isSnippet.Value) {
+                        bool isSnippet = false;
+                        // Snippets are suppressed if user typed namespace
+                        if (!context.IsInNameSpace() && infoSource != null) {
```

Comment:
Is this when we're on the right side of `::`? Should it also exclude snippets on the right side of `@` and `$`?